hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/net/NetUtils.java

```diff
@@ -149,4 +149,28 @@ public static List<Node> getAncestorList(NetworkTopology topology,
     }
     return ancestorList;
   }
+
+  /**
+   * Ensure {@link NetConstants#PATH_SEPARATOR_STR} is added to the suffix of
+   * the path.
+   * @param path path to add suffix
+   * @return the normalised path
+   * If <i>path</i>is empty, then {@link NetConstants#ROOT} is returned
+   */
+  public static String addSuffix(String path) {
+    if (path == null) {
+      return null;
+    }
+
+    int len = path.length();
+    if (len == 0) {
+      return NetConstants.PATH_SEPARATOR_STR;
```

Comment:
In the javadoc you said `` If <i>path</i>is empty, then {@link NetConstants#ROOT} is returned`` but here you are returning ``return NetConstants.PATH_SEPARATOR_STR`` both have different values.

Reply:
Updated the doc to return `PAHT_SEPARATOR_STR` instead.